ci/unit_tests/workspace_delete/main_data/skill-Customer-Care-Sample-Skill.json

```diff
@@ -0,0 +1,1663 @@
+{
```

Comment:
I think smaller dialog json could be sufficient.

Reply:
Done in df287ae7ef51ffd4045060fb964b558e1cca2f57